src/errors/types.rs

```diff
@@ -533,6 +547,9 @@ impl ErrorType {
             Self::UrlSyntaxViolation {..} => "Input violated strict URL syntax rules, {error}",
             Self::UrlTooLong {..} => "URL should have at most {max_length} characters",
             Self::UrlScheme {..} => "URL scheme should be {expected_schemes}",
+            Self::UuidType => "Input should be a string",
```

Comment:
```suggestion
            Self::UuidType => "UUID Input should be a string or UUID object",
```
or something, you'll also want a custom error message for JSON, see the function below.